### PR TITLE
roachtest: fix repro command

### PR DESCRIPTION
**Pull request by tbg**
The link wasn't rendering correctly. This is now fixed and tested.

Release note: None

### CODE REVIEW

File: pkg/cmd/internal/issues/testdata/post/failure-with-url-matching-and-related-issue.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+```
+boom
+```
+<details><summary>Reproduce</summary>
+<p>
+
+
+See: [FooBar README](https://github.com/cockroachdb/cockroach)
+
+<p>Parameters in this failure:
```
Comment (erikgrinaker) [inline pkg/cmd/internal/issues/testdata/post/failure-with-url-matching-and-related-issue.txt:18]: These nested `<p>` tags seem off. In HTML, the second `<p>` will implicitly close the previous one, and so we now have additional spurious `</p>` tags below.

**Review by erikgrinaker — APPROVED** [review]

File: pkg/cmd/internal/issues/testdata/post/failure-with-url-matching-and-related-issue.txt (hunk 1 above)
Comment (tbg) [reply pkg/cmd/internal/issues/testdata/post/failure-with-url-matching-and-related-issue.txt:18]: Good point, fixed in a new commit.

**Comment by tbg** [comment]
Hmm, can't push right now, so the commit isn't there yet. Just a heads up.

**Comment by tbg** [comment]
Now it's there.

**Review by erikgrinaker — APPROVED** [review]
Looks good, thanks!

**Comment by tbg** [comment]
bors r=erikgrinaker

**Pull request merged**